### DISCUSSION TITLE
Google Maps API

### DISCUSSION THREAD

**Discussion by thiagocaparelli** [sepinf-inc/IPED#958] [Q&A]
Hello, 
I have a google API, but when I'm inserting into iped I'm receiving this error.
![Sem título](https://user-images.githubusercontent.com/55517932/153499956-a66835b0-477e-4b64-a953-d135cfeb6de8.png)

**Comment by lfcnassif** [comment]
What is your iped version? Please send the analysis app log.

**Reply by lfcnassif** [reply]
I just tested 3.18.10 and 3.18.12 versions and it is working here.

Edited: Please make sure you created/activated a Google **Maps Javascript API** key and copy it from the following section:

![image](https://user-images.githubusercontent.com/7276994/153515511-edbbdee2-d2a1-4a10-a2c7-95731896c3e7.png)

**Reply by thiagocaparelli** [reply]
It worked.
I had to activate my existent developer key on Google Cloud website to Maps JavaScript API.

Thank you!
